ras/errinjct.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2021 IBM
+# Author: Shirisha Ganta <shirisha.ganta1@ibm.com>
+
+from avocado import Test
+from avocado.utils import process, genio
+from avocado.utils.software_manager import SoftwareManager
+from avocado import skipIf
+
+IS_POWER_NV = 'PowerNV' in genio.read_file('/proc/cpuinfo').rstrip('\t\r\n\0')
+
+class Errinjct(Test):
+    """
+    :avocado: tags=privileged
+    """
+    """
+    The errinjct tool enables RTAS to inject errors on a Power system.
+    Currently, errinjct is only supported on PowerVM systems.
+    """
+    fail_cmd = list()
+
+    def run_cmd(self, cmd):
+        if process.system(cmd, ignore_status=True, sudo=True, shell=True):
+            self.fail_cmd.append(cmd)
+        return
+
+    @staticmethod
+    def run_cmd_out(cmd):
+        return process.system_output(cmd, shell=True,
+                                     ignore_status=True,
+                                     sudo=True).decode("utf-8").strip()
+
+    @skipIf(IS_POWER_NV, "This test is supported on PowerVM environment")
+    def setUp(self):
+        sm = SoftwareManager()
+        if not sm.check_installed("powerpc-utils") and \
+           not sm.install("powerpc-utils"):
+            self.cancel("Fail to install required 'powerpc-utils' package")
+
+    def test_errinjct(self):
+        self.log.info("===Executing errinjct tool====")
+        token = self.run_cmd_out("errinjct open| awk -F '=' '{print $2}'")
+        opt = self.params.get('opt1', default='-a 0 -C 0 --dry-run')
```

Comment:
just a suggestion can we have some meaningful variable name, please , otherwise looks good , 

even i can say we can mux the value like 
runcmd = self.params.get('runcmd', default='corrupted-dcache-start')
run_option = self.params.get('runcmd', default='-a 0 -C 0 --dry-run')


yaml can looks like 
```
value1:
 runcmd : corrupted-dcache-start
 run_option:  '-a 0 -C 0 --dry-run'
 value2:
 runcmd : corrupted-dcache-end
 run_option:  '-a 0 -C 0 --dry-run'
```

using above approach we need to change code any time in future if any new option or run command changes

Reply:
Addressed above comments and made changes to the test case. Please have a look at it once